File: app/src/main/java/design/wendreo/hashisushi/views/policyPrivacy/ActPolicyPrivacy.java
package design.wendreo.hashisushi.views.policyPrivacy;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.webkit.WebView;

import design.wendreo.hashisushi.R;

public class ActPolicyPrivacy extends AppCompatActivity {
	
	@Override
	protected void onCreate ( Bundle savedInstanceState ) {
		super.onCreate ( savedInstanceState );
		setContentView ( R.layout.activity_act_policy_privacy );
		
		WebView webView = findViewById(R.id.wv_content);
		webView.loadUrl("file:///android_asset/politica_privacidade.html");
	}
}

File: app/src/main/java/design/wendreo/hashisushi/views/policyPrivacy/ActPolicyPrivacy.kt
package design.wendreo.hashisushi.views.policyPrivacy

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.webkit.WebView

import design.wendreo.hashisushi.R

class ActPolicyPrivacy : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportActionBar!!.hide()
        setContentView(R.layout.activity_act_policy_privacy)

        val webView = findViewById<WebView>(R.id.wv_content)
        webView.loadUrl("file:///android_asset/politica_privacidade.html")
    }
}
